pr_template.md

```diff
@@ -0,0 +1,8 @@
+#### What's this PR do?
```

Comment:
whoops, I'm sorry - I didn't realize you had included this in your PR, just that your PR wasn't using the template so I created one on `main`
I'm going to delete this one so it doesn't create a duplicate

Reply:
yeah I'd forgotten about the template, and added the template file to the repo after I'd already written my PR description.